### PR TITLE
🌐 Add French translation for `docs/fr/docs/advanced/response-directly.md`

### PR DESCRIPTION
**Pull request by axel584**
Dear translators,
A small translation of an advanced page in :fr: 

Thank you,

Axel

### CODE REVIEW

File: docs/fr/docs/advanced/response-directly.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Et quand vous retournez une `Response`, **FastAPI** la transmet directement.
+
+Elle ne fera aucune conversion de données avec les modèles Pydantic, elle ne convertira pas le contenu en un type quelconque.
+
+Cela vous donne beaucoup de flexibilité. Vous pouvez retourner n'importe quel type de données, surcharger n'importe quelle déclaration ou validation de données.
+
+## Utiliser le `jsonable_encoder` dans une `Response`
+
+Parce que **FastAPI** n'apporte aucune modification à une `Response` que vous retournez, vous devez vous assurer que son contenu est prêt à être utilisé.
```
Comment (raphaelauv) [inline docs/fr/docs/advanced/response-directly.md:28]: prêt à être utilisé (serializable).

File: docs/fr/docs/advanced/response-directly.md [hunk 2]
```diff
@@ -1,63 +1,63 @@
-# Return a Response Directly
+# Retourner une réponse directement
```
Comment (frabc) [inline docs/fr/docs/advanced/response-directly.md:1]: For me the sentence `Renvoyer directement une réponse` sounds better.

File: docs/fr/docs/advanced/response-directly.md [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
 
-When you create a **FastAPI** *path operation* you can normally return any data from it: a `dict`, a `list`, a Pydantic model, a database model, etc.
+Lorsque vous créez une *opération de chemins* **FastAPI**, vous pouvez normalement retourner n'importe quelle donnée : un `dict`, une `list`, un modèle Pydantique, un modèle de base de données, etc.
 
-By default, **FastAPI** would automatically convert that return value to JSON using the `jsonable_encoder` explained in [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank}.
+Par défaut, **FastAPI** convertirait automatiquement cette valeur de retour en JSON en utilisant le `jsonable_encoder` expliqué dans [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank}.
 
-Then, behind the scenes, it would put that JSON-compatible data (e.g. a `dict`) inside of a `JSONResponse` that would be used to send the response to the client.
+Ensuite, dans les coulisses, il mettra ces données compatibles JSON (par exemple un `dict`) à l'intérieur d'un `JSONResponse` qui sera utilisé pour envoyer la réponse au client.
```
Comment (frabc) [inline docs/fr/docs/advanced/response-directly.md:7]: I will leave the word `JSON-compatible` in place of `compatibles-JSON`.

Comment (frabc) [reply docs/fr/docs/advanced/response-directly.md:7]: I would have translated `behind the scenes` by the word `en arrière-plan` or `en coulisses`.

File: docs/fr/docs/advanced/response-directly.md [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
+Par exemple, vous ne pouvez pas mettre un modèle Pydantic dans une `JSONResponse` sans d'abord le convertir en un `dict` avec tous les types de données (comme `datetime`, `UUID`, etc) convertis en types compatibles avec JSON.
 
-For those cases, you can use the `jsonable_encoder` to convert your data before passing it to a response:
+Pour ces cas, vous pouvez spécifier un appel à `jsonable_encoder` pour convertir vos données avant de les passer à une réponse :
 
-```Python hl_lines="6-7  21-22"
-{!../../../docs_src/response_directly/tutorial001.py!}
+``Python hl_lines="6-7 21-22"
+{ !../../../docs_src/response_directly/tutorial001.py!}
 ```
```
Comment (frabc) [inline docs/fr/docs/advanced/response-directly.md:36]:
```
```Python hl_lines="6-7  21-22"
{!../../../docs_src/response_directly/tutorial001.py!}
```

There is a missing ` before the word python. 
There is also a lack or too much whitespace.

File: docs/fr/docs/advanced/response-directly.md [hunk 5]
```diff
@@ -1,63 +1,63 @@
...
+Disons que vous voulez retourner une réponse <a href="https://en.wikipedia.org/wiki/XML" class="external-link" target="_blank">XML</a>.
 
-You could put your XML content in a string, put it in a `Response`, and return it:
+Vous pouvez mettre votre contenu XML dans une chaîne de caractères, la placer dans une `Response`, et la retourner :
 
-```Python hl_lines="1  18"
-{!../../../docs_src/response_directly/tutorial002.py!}
+``Python hl_lines="1 18"
+{ !../../../docs_src/response_directly/tutorial002.py!}
 ```
```
Comment (frabc) [inline docs/fr/docs/advanced/response-directly.md:55]: Same here:

```
```Python hl_lines="1  18"
{!../../../docs_src/response_directly/tutorial002.py!}
```

File: docs/fr/docs/advanced/response-directly.md (hunk 1 above)
Comment (frabc) [reply docs/fr/docs/advanced/response-directly.md:28]: `(sérialisable).`

**Review by cfraboulet — COMMENTED** [review]
Hi @axel584 😊 

Some comments about your PR! 

Moreover, I don't know why but the CI does not pass.

File: docs/fr/docs/advanced/response-directly.md (hunk 1 above)
Comment (frabc) [reply docs/fr/docs/advanced/response-directly.md:28]: `(sérialisable).`

Comment (axel584) [reply docs/fr/docs/advanced/response-directly.md]: I correct this, if it's ok for you, please, close the comment. Thanks.

File: docs/fr/docs/advanced/response-directly.md (hunk 2 above)
Comment (axel584) [reply docs/fr/docs/advanced/response-directly.md]: I correct this, if it's ok for you, please, close the comment. Thanks.

File: docs/fr/docs/advanced/response-directly.md (hunk 5 above)
Comment (axel584) [reply docs/fr/docs/advanced/response-directly.md:55]: I correct this, if it's ok for you, please, close the comment. Thanks.

File: docs/fr/docs/advanced/response-directly.md (hunk 4 above)
Comment (axel584) [reply docs/fr/docs/advanced/response-directly.md:36]: I correct this, if it's ok for you, please, close the comment. Thanks.

File: docs/fr/docs/advanced/response-directly.md (hunk 3 above)
Comment (axel584) [reply docs/fr/docs/advanced/response-directly.md]: I correct this, if it's ok for you, please, close the comment. Thanks.

**Review by cfraboulet — APPROVED** [review]
LGTM 👍🏻 
Well done @axel584 🚀 😊

**Review by emmrichard — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @axel584! :rocket: 

And thanks for the reviews @raphaelauv, @emmrichard, @frabc :cake:

**Pull request merged**